### ISSUE TITLE
Lock the teleportation functions rotation to the hmd.

### ISSUE THREAD

**Issue by EnderActually** [GodotVR/godot-vr-common#651]
Is there any way to lock the teleportation functions rotation to the headset?
If there isn't any way to do that, it would be nice to add that as a setting.

Another thing I noticed, is that the teleportation rotation is affected by the hands rotation, fixing that would also be nice.

**Comment by vigneshiamvk** [comment]
Yes, it is disorienting when you teleport and your orientation is not what you expected it to be.
